gears/src/context/mod.rs

```diff
@@ -10,20 +12,31 @@ pub mod tx;
 
 pub trait QueryableContext<DB, SK> {
     /// Fetches an immutable ref to a KVStore from the MultiStore.
-    fn kv_store(&self, store_key: &SK) -> KVStore<'_, PrefixDB<DB>>;
+    fn kv_store(&self, store_key: &SK) -> Store<'_, PrefixDB<DB>>;
```

Comment:
As I understand you've used the method `kv_store` and `infallible_store` in params keepers. 

Where we can use it except params? Can we rewrite this part?

Reply:
Not quite understand this question. `KvStore` is mainly used for params.